client/components/dashboard/dashboard-directive.js

```diff
@@ -85,6 +85,19 @@ explorer.components.dashboard.DashboardDirective = function() {
       }
 
       /** @export */
+      $scope.getSelectedClass = function(container) {
+        if (container.state().selected) {
+          if (explorerStateService.widgets.selected) {
+            return 'pk-container-selected-partial';
```

Comment:
That is correct.  Renamed to -implicit instead of -partial, and added a clarifying comment to the style.